cypress/integration/Train/ScenarioAndTags.spec.js

```diff
@@ -79,7 +79,7 @@ describe('Scenario and Tags - Train Dialog', () => {
         // reload
         // Verify edited tags and description are still on dialog
         editDialogModal.TypeScenario('Edited Test Scenario')
-        editDialogModal.AddTag('TagY')
+        editDialogModal.AddTags('TagY')
```

Comment:
Hmm, less important that the other things about test structure but if you're renaming this to be plural I would expect the method to take an array or multiple arguments.
Meaning you should haven't change the type of arguments depending on whether you want to add 1 or more.

This means the method should take:
```
addTags('tag01', 'tag02', 'tag03', ...)
addTags(['tag01', 'tag02', 'tag03'])
```
The latter form is more natural but I would be ok with former

Reply:
This function does take either a single parameter or an array. Internally it converts to an array if a single string was passed in, and then proceeds to process it. So either of these calling syntaxes work:

> addTags('tag01')
addTags(['tag01', 'tag02', 'tag03'])
>

Reply:
Yes, I know I see the line: `if (!Array.isArray(tags)) tags = [tags]`
https://github.com/Microsoft/ConversationLearner-UI/pull/1030/files#diff-bff231d2b160a6454f2005358f643af4R47

> you should haven't change the type of arguments

Yes, going from string to array would be considered changing the type.

Reply:
Working on it...

Reply:
Updates to `cease changing the type` in this case and elsewhere have been made.